src/platform/remote/comfyui/history/types/historyV1Types.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @fileoverview History V1 types - Public interface used throughout the app
+ * @module platform/remote/comfyui/history/types/historyV1Types
+ *
+ * These types represent the V1 history format that the application expects.
+ * Both desktop (direct V1 API) and cloud (V2 API + adapter) return data in this format.
+ */
+
+import type { HistoryTaskItem, TaskPrompt } from '@/schemas/apiSchema'
```

Comment:
Moving these into here would trigger a larger refactor of queueStore and several other files. Let me know if you want me to push through on that as well.